app/mailers/user_mailer.rb

```diff
@@ -9,21 +9,26 @@ class UserMailer < ActionMailer::Base
   end
 
   # checks the status of the current reservation and sends the appropriate email
-  # receipt forces a check out receipt to be sent
-  def reservation_status_update(reservation, receipt = false) # rubocop:disable all
+  # force forces a check out receipt to be sent OR an approved request email,
```

Comment:
I'm not thrilled with this; it's making the method a little difficult to understand and definitely makes it tricky to modify in the future. Here's an alternative suggestion - default `force` to an empty string and use that as a forced `@status` (allowing you to send any of the pre-defined statuses, overriding the current status). Therefore to send a receipt you'd call `reservation_status_update(res, 'receipt')` and to send the approval e-mail you'd call `reservation_status_update(res, 'request approved')`. Obviously you'd need some logic to infer from `receipt` whether nor not it was the checkout or checkin receipt, but this to me seems cleaner. Thoughts?